autoload/neomake/makers/tex.vim

```diff
@@ -0,0 +1,28 @@
+" vim: ts=4 sw=4 et
+
+function! neomake#makers#tex#EnabledMakers()
+    if neomake#utils#Exists('chktex')
```

Comment:
This would work but probably isn't necessary (unless I'm missing something). Makers that fail the `neomake#utils#Exists` check are filtered out before they are invoked. The only reason we had this function in the `neomake#makers#python#EnabledMakers` function is just because flake8 depended on pep8 and pyflakes, which would have resulted in redundant runs.

Normally, all that is required is to return a straight list of all the checkers that are defined, and only the makers the user has installed will run.